### PR TITLE
[generel] Added a Licensing Model for the Importer (major)

### PR DESCRIPTION
**Pull request by SirGibihm**
The importer shall be run unter the AGLv3 License. This should be checked + verified by @codeofsumit and @dredav

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,3 +51,23 @@ To contribute:
...
 7. Create a Pull Request
+
+# Licensing
+
+Usage of this codebase is permitted according to the GNU Affero General Public License v3 (AGPL-3.0).
+**THIS PARAGRAPH GIVES ONLY AN INCOMPLETE AND ROUGH OVERVIEW. READ THE LICENSE BEFORE USING**
+
+- You are allowed to:
+  - Use this codebase in any way you want including commercially
+  - Modify and distribute the Codebase in any way ou want
```
Comment (codeofsumit) [inline README.md:62]:
```suggestion
  - Modify and distribute the codebase in any way you want
```

File: README.md [hunk 2]
```diff
@@ -51,3 +51,23 @@ To contribute:
 5. Write a parser in `src/brokers` to parse that content - see `src/brokers/comdirect.js` for inspiration
 6. Add and run all tests `npm t`
 7. Create a Pull Request
+
+# Licensing
+
+Usage of this codebase is permitted according to the GNU Affero General Public License v3 (AGPL-3.0).
+**THIS PARAGRAPH GIVES ONLY AN INCOMPLETE AND ROUGH OVERVIEW. READ THE LICENSE BEFORE USING**
+
+- You are allowed to:
```
Comment (codeofsumit) [inline README.md:60]:
```suggestion
- You are permitted to:
```

File: README.md [hunk 3]
```diff
@@ -51,3 +51,23 @@ To contribute:
 5. Write a parser in `src/brokers` to parse that content - see `src/brokers/comdirect.js` for inspiration
 6. Add and run all tests `npm t`
 7. Create a Pull Request
+
+# Licensing
+
+Usage of this codebase is permitted according to the GNU Affero General Public License v3 (AGPL-3.0).
+**THIS PARAGRAPH GIVES ONLY AN INCOMPLETE AND ROUGH OVERVIEW. READ THE LICENSE BEFORE USING**
```
Comment (codeofsumit) [inline README.md:58]:
```suggestion
**HERE'S AN INCOMPLETE SUMMARY. READ THE FULL LICENSE BEFORE USING**
```

File: README.md [hunk 4]
```diff
@@ -51,3 +51,23 @@ To contribute:
...
+
+- You are allowed to:
+  - Use this codebase in any way you want including commercially
+  - Modify and distribute the Codebase in any way ou want
+- You MUST:
+  - respect the Copyright which is held by the maintainers of this project
```
Comment (codeofsumit) [inline README.md:64]:
```suggestion
  - respect the copyright which is held by the maintainers of this project
```

File: README.md [hunk 5]
```diff
@@ -51,3 +51,23 @@ To contribute:
...
+  - You MUST disclose all changes you made to the code and also publish any code directly based on this code
```
Comment (codeofsumit) [inline README.md:65]:
```suggestion
  -  disclose all changes you made to the code and also publish any code directly based on this code
```

File: README.md [hunk 6]
```diff
@@ -51,3 +51,23 @@ To contribute:
...
+  - You MUST give attribution to the orignal maintainers when using this project (especially when used commercially)
```
Comment (codeofsumit) [inline README.md:66]:
```suggestion
  - give attribution to the orignal maintainers when using this project (especially when used commercially)
```

File: README.md [hunk 7]
```diff
@@ -51,3 +51,23 @@ To contribute:
...
+- You can not
+  - Sublicense this codebase
+  - expect any Liability, Warranty or similar by the original Maintainers. However, we want to improve
```
Comment (codeofsumit) [inline README.md:69]:
```suggestion
  - expect any liability, warranty or similar by the original maintainers. However, we want to improve
```

File: README.md (hunk 3 above)
Comment (codeofsumit) [reply README.md:58]: Let's add a link here to the full license.

File: LICENSE [hunk 8]
```diff
@@ -0,0 +1,661 @@
...
+get its source.  For example, if your program is a web application, its
+interface could display a "Source" link that leads users to an archive
+of the code.  There are many ways you could offer source, and different
+solutions will be better for different programs; see section 13 for the
+specific requirements.
+
+  You should also get your employer (if you work as a programmer) or school,
+if any, to sign a "copyright disclaimer" for the program, if necessary.
+For more information on this, and how to apply and follow the GNU AGPL, see
+<https://www.gnu.org/licenses/>.
```
Comment (codeofsumit) [inline LICENSE:661]: 😍  amazing!

**Review by codeofsumit — CHANGES_REQUESTED** [review]
Great great great!!!

**Comment by SirGibihm** [comment]
@dredav I implemented Sumits change requests. If you don't find anything you would like to see changed please merge

File: README.md [hunk 9]
```diff
@@ -51,3 +51,23 @@ To contribute:
...
+- You MUST:
+  - respect the copyright which is held by the maintainers of this project
+  - disclose all changes you made to the code and also publish any code directly based on this code
+  - give attribution to the orignal maintainers when using this project (especially when used commercially)
+- You can not
+  - Sublicense this codebase
+  - expect any liability, warranty or similar by the original maintainers. However, we want to improve
+    this project as much as possible so feedback is heard!
+
+See also the following image, taken from [TL;DR Legal](https://www.tldrlegal.com/l/agpl3).
```
Comment (dredav) [inline README.md:72]: Oh nice service. Good to know it.

File: README.md [hunk 10]
```diff
@@ -51,3 +51,23 @@ To contribute:
 5. Write a parser in `src/brokers` to parse that content - see `src/brokers/comdirect.js` for inspiration
 6. Add and run all tests `npm t`
 7. Create a Pull Request
+
+# Licensing
+
+Usage of this codebase is permitted according to the GNU Affero General Public License v3 (AGPL-3.0).
+**HERE'S AN INCOMPLETE SUMMARY. READ THE [FULL LICENSE](LICENSE) BEFORE USING**
```
Comment (dredav) [inline README.md:58]: I thinks it's better to add a new line for a better readability.

```suggestion

**HERE'S AN INCOMPLETE SUMMARY. READ THE [FULL LICENSE](LICENSE) BEFORE USING.**
```

**Review by dredav — COMMENTED** [review]
Could you also change the license in the `package.json`?

File: README.md [hunk 11]
```diff
@@ -51,3 +51,23 @@ To contribute:
...
+
+- You are permitted to:
+  - Use this codebase in any way you want including commercially
+  - Modify and distribute the codebase in any way you want
+- You MUST:
+  - respect the copyright which is held by the maintainers of this project
```
Comment (dredav) [inline README.md:64]: Also I would recomend to start each point with an capital letter

**Comment by dredav** [comment]
This is a great and important change! Thanks for adressing it @SirGibihm. I only requested some minor changes.

**Comment by SirGibihm** [comment]
@dredav Worked in your comments, especially the ones regarding the package-lock

**Review by dredav — APPROVED** [review]

**Review by dredav — APPROVED** [review]

**Comment by dredav** [comment]
Mergin is blocked because @codeofsumit has requested changes previously. Waiting for a second approval.

**Review by codeofsumit — APPROVED** [review]

**Comment by carstenhag** [comment]
Nice move, I think the AGPL is a good fit.  

FYI though, tresor.one, import.tresor.one etc must now disclose the usage of this lib :)

**Pull request merged**